### ISSUE TITLE
Execute selection and then move cursor to next line of selection

### ISSUE THREAD

**Issue by zippeurfou** [julienr/vim-cellmode#25]
It would be nice to have an alternative to `CTRL + C` that goes to the next line of the selection.

**Comment by julienr** [comment]
Thanks for the suggestion !

So you would like to run a selection line by line ? `Ctrl-C` currently runs the whole selection (doesn't it ?), so you can't really go to the next line of selection after that.

Or did I misunderstand something ?

**Comment by zippeurfou** [comment]
I would like to run current selection and then go to the next line (down) instead of going back to the beginning of the selection as it currently dows. The same way nvim-R does `\d`
https://github.com/jalvesaq/Nvim-R/blob/master/doc/Nvim-R.txt#L382
Current behavior https://cl.ly/b23166eabab6
And what behavior I'd like (example using R): https://cl.ly/367b972d2b35
The behavior is different as after executing the selection it goes to the next line. Let me know if this is not clear enough.

**Comment by julienr** [comment]
Ah ok, make sense ! Then yes, I agree it would be a nice addition. I can't promise anything because I'm quite busy currently, but if you feel like doing a PR, please go ahead :)

**Comment by zippeurfou** [comment]
Thanks @julienr, I wonder if we could just do a shortcut somehow that invoke `CTRL-C` as a quick hack at least.

**Comment by julienr** [comment]
Yeah I guess it would be a matter of writing another function that calls [`RunTmuxPythonChunk `](https://github.com/julienr/vim-cellmode/blob/master/ftplugin/python/cellmode.vim#L261) and then goes to the next line. I'm actually pretty bad at vimscript, so I don't even know how to go to the next line, but something like simulating a `j` should do.
